PFCalEE/userlib/include/HGCSSSimHit.hh

```diff
@@ -17,7 +17,7 @@
 
 
 //for hexagons: side size.
-static const float CELL_SIZE_X=6.4;//2.5;//mm
+static const float CELL_SIZE_X=6.496345; //2.5;//mm
```

Comment:
The pad pitch was quoted to me with 5 significant digits and the sensor 
size to 6 significant digits.

On 4/5/2016 2:45 AM, amagnan wrote:
>
> In PFCalEE/userlib/include/HGCSSSimHit.hh 
> <https://github.com/pfs/PFCal/pull/17#discussion_r58498668>:
>
> > @@ -17,7 +17,7 @@
> >
> >
> >  //for hexagons: side size.
> > -static const float CELL_SIZE_X=6.4;//2.5;//mm
> > +static const float CELL_SIZE_X=6.496345; //2.5;//mm
>
> will it be that precise in real life ;) ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/pfs/PFCal/pull/17/files/10d68cea58df7a3601955203a93d7cc949b732a4#r58498668>
>